### ISSUE TITLE
Conditions Formatting Not 

### ISSUE THREAD

**Issue by adamnimmo** [bhavesh-jadav/Advance-Card#16]
Hello, I seem to be having an issue with conditions formatting, I have selected placed a measure within the Data Field (59) and I have a target measure within the Condition Field (45), when I turn on Conditions within the format pane I am not getting the behavior I am expecting.

Based on the documentation I interpreted this as a auto field which will compare Data v Conditions and format properly. Both of my measures are formatted as a whole number w/o decimal places.

If I enter a static value within the Conditions Value box I do get the result expected. 

Did I misunderstand the documentation or am I doing something wrong? 

Thank you for your help

![cards](https://user-images.githubusercontent.com/25329556/45319816-b87d3280-b4f5-11e8-9b04-3a2c2ebdf385.PNG)
![conditions](https://user-images.githubusercontent.com/25329556/45319819-bb782300-b4f5-11e8-91f5-aedb43c00fb1.PNG)

![image](https://user-images.githubusercontent.com/25329556/45319831-c6cb4e80-b4f5-11e8-9341-b909e420e5fa.png)

**Comment by bhavesh-jadav** [comment]
Hi @adamnimmo 

When you drop field in **Condition field**, that value will be used as base value for comparison and will be compared with **Value** in **Conditions** which from what I can see from screenshot, you have kept it empty.

If you want to compare **MilOOB** with **OOB_Target** then you can create a measure which might look like below.
```
Measure = IF([MilOOB] > [OOB_Target], 1, 0)
```
Then drop this Measure into conditions fields and set conditions to 2 and other settings as follow.
If Measure value is 1 then red and If it is 0 then set it as green something like that.
You will specify 1 and 0 in **Value** of Conditions

In short, comparison happens between either Datafield value and static value if no Condition field is specified or between Condition field and static value if condition field is specified. So if you want to compare both of them, create a measure which returns different static value based on conditions  and use that as conditional formatting.

**Comment by adamnimmo** [comment]
Brilliant @bhavesh-jadav I figured I was interpreting something incorrectly. 

Thank you very much for the reply and for creating this outstanding visual.
